boards/nrf52dk/Makefile.include

```diff
@@ -10,7 +10,7 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
 export JLINK_DEVICE := nrf52
 
 # special options when using SoftDevice
-ifneq (,$(filter nordic_softdevice_ble,$(USEPKG)))
+ifeq (,$(filter nordic_softdevice_ble,$(USEPKG)))
```

Comment:
This change works when nordic_softdevice_ble is necessary, but for instance the hello-world example doesn't work... This needs to be defined somewhere else (after `USEPKG += nordic_softdevice_ble` is declared), but I don't have in mind right now where it's suitable.

Reply:
As tested with @kYc0o, this check cannot work as `nordic_softdevice_ble` is added to `USEPKG` only during dependency resolution with happens after including `Makefile.include`.